sdk/storage/azure-storage-blob-changefeed/src/main/java/com/azure/storage/blob/changefeed/implementation/models/BlobChangefeedEvent.java

```diff
@@ -0,0 +1,212 @@
+// Copyright (c) Microsoft Corporation. All rights reserved.
+// Licensed under the MIT License.
+
+package com.azure.storage.blob.changefeed.implementation.models;
+
+import com.azure.storage.blob.changefeed.models.BlobChangefeedEventType;
+import com.azure.storage.internal.avro.implementation.AvroConstants;
+import com.azure.storage.internal.avro.implementation.schema.AvroSchema;
+import com.azure.storage.internal.avro.implementation.schema.primitive.AvroNullSchema;
+
+import java.time.OffsetDateTime;
+import java.util.Map;
+import java.util.Objects;
+
+/**
+ * This class contains properties of a BlobChangefeedEvent.
+ */
+public class BlobChangefeedEvent implements com.azure.storage.blob.changefeed.models.BlobChangefeedEvent {
```

Comment:
Should I call this ...EventImpl? Analyze Step is mad at me for keeping the same name

Reply:
Noooooo https://octoperf.com/blog/2016/10/27/impl-classes-are-evil/ .

Maybe InternalBlobChangefeedEvent ?

Reply:
haha, I'll go with Internal

Reply:
renamed to Internal